### PR TITLE
Develop -> master

### PR DESCRIPTION
**Pull request by sergei-maertens**

### CODE REVIEW

File: Changelog.rst [hunk 1]
```diff
@@ -2,6 +2,21 @@
 Changelog
 =========
 
+1.5.0
+-----
+
+* Dropped support for old Python/Django versions.
+* Added support for ``NullBooleanField`` -- thanks to @ashwch
```
Comment (merwok) [inline Changelog.rst]: Need note for the new guarantee that the values attribute preserves definition order.

File: README.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ Order and sanity for django model choices.
 ------------------------------------------------------
 
 Django choices provides a declarative way of using the choices_ option on django_
-fields.
+fields. Read the full `Documentation`_ on ReadTheDocs.
```
Comment (merwok) [inline README.rst]: Marked-up word is Documentation here but the target says docs.

File: Changelog.rst (hunk 1 above)
Comment (sergei-maertens) [inline Changelog.rst]: Was added via my release branch, going through CI now

File: README.rst (hunk 2 above)
Comment (sergei-maertens) [inline README.rst]: thanks, I messed up there.

**Comment by sergei-maertens** [comment]
@merwok @ashwch release to PyPI: https://pypi.python.org/pypi/django-choices

**Pull request merged**